compiler/noirc_evaluator/src/ssa/plonky2_gen/mod.rs

```diff
@@ -345,6 +347,16 @@ impl Builder {
                 Ok(_) => (),
             }
         }
+        let mut next_param_idx: usize = 0;
+        for (_, typ, vis) in main_function_signature.0 {
```

Comment:
I thought that the first _ might be name, in which case we could drop the parameter_names argument, but I checked and it's HirPattern.

Very strange approach to method signature indeed, but I guess they have good reasons for not including the names (perhaps to use it as part of the type of the function).

No action required.